app/controllers/katello/api/v2/distributions_controller.rb

```diff
@@ -1,38 +0,0 @@
-module Katello
-  class Api::V2::DistributionsController < Api::V2::ApiController
```

Comment:
If we are removing this, we need to also remove the routes related to it, further, we need to ask ourselves if this is breaking our API contract and if we need to rev our versioning in light of.